src/middlewares/validators/handlers.js

```diff
@@ -8,6 +10,28 @@ function handleErrorDetails(error, next) {
   return next(new ValidationError(details));
 }
 
+function validateSkipLimit(req, res, next) {
+  const { skip, limit } = req.query;
+
+  const schema = Joi.object().keys({
+    limit: Joi.number().integer().positive(),
+    skip: Joi.number().integer().positive()
+  });
+
+  const result = schema.validate({ skip, limit });
```

Comment:
destruct error instead of keeping in result, then accessing with dot

Reply:
also pass the whole req to validate and validation options as in other validators